web/src/settings_users.ts

```diff
@@ -480,6 +480,17 @@ function active_create_table(active_users: number[]): void {
     $("#admin_users_table").show();
 }
 
+function handle_clear_button_for_users($tbody: JQuery): void {
+    const $container = $tbody.closest(".user-settings-section");
+    $container.on("click", ".clear-filter", (e) => {
+        e.stopPropagation();
+        e.preventDefault();
+        const $filter = $container.find(".search");
+        set_text_search_value($tbody, "");
+        $filter.trigger("input");
+    });
```

Comment:
I don't understand the construction of this handler -- it seems like you might register it several times.

Reply:
Oh, I guess it is only called once. The `handle_` naming scheme for these functions are a bit weird, but that's not new in this PR.